system/lib/emmalloc.cpp

```diff
@@ -62,6 +63,9 @@ extern "C"
 // than this will yield an allocation with this much alignment.
 #define MALLOC_ALIGNMENT 8
```

Comment:
How about

```
#define MALLOC_ALIGNMENT alignof(max_align_t)
_Static_assert(alignof(max_align_t) == 8, "max_align_t must be correct");
```

Note that in C++ you can do `alignof` rather than `__alignof__`

Reply:
Sure.